lib/oli/resources/explanation_strategy.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Oli.Resources.ExplanationStrategy do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  def types() do
+    [
+      :none,
+      :after_max_resource_attempts_exhausted,
+      :after_set_num_attempts
+    ]
+  end
+
+  @derive Jason.Encoder
+  @primary_key false
+  embedded_schema do
+    field :type, Ecto.Enum,
+      values: [
+        :after_max_resource_attempts_exhausted,
```

Comment:
It seems like `:none` be included here?

Reply:
Actually I had just forgotten to remove `:none` from the above `types()`. I opted to model no policy as `nil` so that this wouldn't require a migration.

If you think that explicitly modeling this and writing a migration is a better way to go though, I can do that